cli/tests/integrations/test_service.py

```diff
@@ -99,28 +99,21 @@ def test_service_inactive_and_completed():
 
 
 def test_log():
-    with package(
-        'cassandra',
-        deploy=True,
-        args=['--package-version=1.0.25-3.0.10']
-    ):
+    with package('hello-world', deploy=True):
```

Comment:
Are you ok with depending on a non-fixed version of `hello-world`?

Reply:
There were quite some problems with this, tests checking the `package describe` output of unbounded packages. Here I left it as is because it's a very basic test. Deploy the app and see if you can get the logs, it doesn't check for specific messages or so.

So I assumed this is safe, and this would let us catch potential problems of future hello-world releases.